### PR TITLE
Workout page

### PR DESCRIPTION
**Pull request by Karumu05**

### CODE REVIEW

File: app/(tabs)/plan/_layout.tsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+      <Stack.Screen
+        name="home"
+        options={{
+          headerTitle: 'Daily Plan',
+        }}
+      />
+      <Stack.Screen
+        name="workout"
+        options={{
+          headerTitle: 'Todays Workout',
```
Comment (elenatrr) [inline app/(tabs)/plan/_layout.tsx]: Today's Workout

**Review by Siffster85 — CHANGES_REQUESTED** [review]
button.tsx need to remove the comment on line 32
addExercise.tsx has a text field of test on line 6, is this needed?

File: app/(tabs)/plan/index.tsx [hunk 2]
```diff
@@ -1,18 +1,5 @@
...
-        <Text>Go To Workout</Text>
-      </Link>
-    </View>
-  )
-}
-
-export default plan;
\ No newline at end of file
+export default function index () {
+  return <Redirect href="/plan/home"/>
```
Comment (elenatrr) [inline app/(tabs)/plan/index.tsx]: can get rid of index page

File: app/(tabs)/plan/exercises/[id].tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Stack, router, useLocalSearchParams } from "expo-router";
+import { Text, View } from "react-native";
+
+export default function currentExercise () {
+
```
Comment (elenatrr) [inline app/(tabs)/plan/exercises/[id].tsx:6]: extra line

File: app/(tabs)/plan/exercises/[id].tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    const {id, title} = useLocalSearchParams()
+
```
Comment (elenatrr) [inline app/(tabs)/plan/exercises/[id].tsx]: extra line

**Review by elenatrr — CHANGES_REQUESTED** [review]

**Review by elenatrr — APPROVED** [review]

**Review by Siffster85 — APPROVED** [review]

**Pull request merged**